lib/node_modules/@stdlib/math/base/special/cinvf/test/test.js

```diff
@@ -0,0 +1,446 @@
+
+	re = tinyPositiveRealComponents.re;
+	im = tinyPositiveRealComponents.im;
+	qre = tinyPositiveRealComponents.qre;
+	qim = tinyPositiveRealComponents.qim;
+
+	for ( i = 0; i < re.length; i++ ) {
+		q = cinvf( new Complex64( re[ i ], im[ i ] ) );
+
+		if ( real( q ) === qre[ i ] ) {
+			t.strictEqual( real( q ), qre[ i ], 'returns expected real component' );
+		} else {
+			delta = absf( real( q ) - qre[ i ] );
+			tol = 3 * EPS * absf( qre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re[i]+'+ '+im[i]+'i. real: '+real( q )+'. expected: '+qre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( imag( q ) === qim[ i ] ) {
+			t.strictEqual( imag( q ), qim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = absf( imag( q ) - qim[ i ] );
+			tol = 3 * EPS * absf( qim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re[i]+'+ '+im[i]+'i. imag: '+imag( q )+'. expected: '+qim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function may produce very large values for very small inputs', function test( t ) {
+	var huge = 1e37;
+	var tiny = -1e37;
+	var v;
+
+	v = cinvf( new Complex64( FLOAT32_SMALLEST_NORMAL, FLOAT32_SMALLEST_NORMAL ) ); // eslint-disable-line max-len
+	t.ok( real( v ) > huge, 'real component is very large' );
+	t.ok( imag( v ) < tiny, 'imaginary component is very large and negative' );
+
+	v = cinvf( new Complex64( -FLOAT32_SMALLEST_NORMAL, FLOAT32_SMALLEST_NORMAL ) ); // eslint-disable-line max-len
+	t.ok( real( v ) < tiny, 'real component is very large and negative' );
+	t.ok( imag( v ) < tiny, 'imaginary component is very large and negative' );
+
+	v = cinvf( new Complex64( -FLOAT32_SMALLEST_NORMAL, -FLOAT32_SMALLEST_NORMAL ) ); // eslint-disable-line max-len
+	t.ok( real( v ) < tiny, 'real component is very large and negative' );
+	t.ok( imag( v ) > huge, 'imaginary component is very large and positive' );
+
+	v = cinvf( new Complex64( FLOAT32_SMALLEST_NORMAL, -FLOAT32_SMALLEST_NORMAL ) ); // eslint-disable-line max-len
+	t.ok( real( v ) > huge, 'real component is very large and positive' );
+	t.ok( imag( v ) > huge, 'imaginary component is very large and positive' );
+
+	v = cinvf( new Complex64( 0.0, FLOAT32_SMALLEST_NORMAL ) );
+	t.strictEqual( real( v ), 0.0, 'real component is 0' );
+	t.ok( imag( v ) < tiny, 'imaginary component is very large and negative' );
+
+	v = cinvf( new Complex64( 0.0, -FLOAT32_SMALLEST_NORMAL ) );
+	t.strictEqual( real( v ), 0.0, 'real component is 0' );
+	t.ok( imag( v ) > huge, 'imaginary component is very large and positive' );
+
+	v = cinvf( new Complex64( FLOAT32_SMALLEST_NORMAL, 0.0 ) );
+	t.ok( real( v ) > huge, 'real component is very large and positive' );
+	t.strictEqual( imag( v ), 0.0, 'imaginary component is 0' );
+
+	v = cinvf( new Complex64( -FLOAT32_SMALLEST_NORMAL, 0.0 ) );
+	t.ok( real( v ) < tiny, 'real component is very large and negative' );
+	t.strictEqual( imag( v ), 0.0, 'imaginary component is 0' );
```

Comment:
I think it would be more better if we use something like:
```bash
returns expected value.
```
for these tests, and those in `test.native.js`.